### PR TITLE
Bill/raceoptimization/calculations

### PR DESCRIPTION
**Pull request by bill-luu**
(Not ready for review yet, just want to get some thoughts on how we're implementing the calcs)

### CODE REVIEW

File: server/scripts/amqp.js [hunk 1]
```diff
@@ -49,6 +46,36 @@ amqp.connect(config.rabbitmq.host)
...
+          // Process Lap data
+          // Want to capture when button is released
+          // TODO: Swap VolumeDown with lap button when it's ready
+          if (!jsonObj.DriverControls.VolumeDown
+            && volumeDown) {
+            db.between(lastLapTimestamp, jsonObj.TimeStamp)
+               .then((allPackets) => {
+                 let averagePackCurrent = rc.getAveragePackCurrent(allPackets);
+                 let amphours = jsonObj.Battery.PackAmphours;
+                 let batterysecondsremaining = rc.getSecondsRemainingUntilChargedOrDepleted(averagePackCurrent, amphours);
```
Comment (Mavestri) [inline server/scripts/amqp.js]: Average pack current uses camelcase while amphours and batterysecondsremaining uses lowercase, was this intended?

Comment (bill-luu) [reply server/scripts/amqp.js]: Nope, good catch!

**Comment by bill-luu** [comment]
Continued in #134

File: server/scripts/amqp.js (hunk 1 above)
Comment (JaydonFernandes) [reply server/scripts/amqp.js:59]: lgtm

**Pull request closed without merging**